Game/MyProject/Common.xcodeproj/xcshareddata/xcschemes/Game.xcscheme

```diff
@@ -46,7 +46,8 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       launchStyle = "0"
-      useCustomWorkingDirectory = "NO"
+      useCustomWorkingDirectory = "YES"
+      customWorkingDirectory = "/Users/tamasb/Documents/Dev/RacingGame/Game/MyProject/Game"
```

Comment:
This should most definitely NOT be on Git. Try to find a decent gitignore, this will break the other guys' workspaces.